Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```diff
@@ -163,6 +163,21 @@ configuration:
   - Indicator of Misconfiguration
   - Indicator of Attack
   - Mobile Detection
+- defaultvalue: 'New,In progress,True positive,False positive,Reopened,Ignored'
```

Comment:
This is the current behavior?

Reply:
Yes, since it up until now, the detections with these statuses was always reopens, the default value matches the current behavior.
The other incident types that supports mirroring CrowdStrike Falcon Incident, CrowdStrike Falcon IDP Detection, and CrowdStrike Falcon Mobile Detection have a subset of these statuses, so it will catch in those cases as well.